src/main/java/Duke.java

```diff
@@ -1,10 +1,206 @@
+import  java.util.Scanner;
+
 public class Duke {
+
+    public static final String BYE_MESSAGE = "\n\tSee you! Have a nice day!\n";
+    public static final String UNIDENTIFIED_MESSAGE = "\n\tSorry! :( I don't understand.\n";
+    public static final String DEADLINE_IDENTIFIER = "/by ";
+    public static final String EVENT_IDENTIFIER = "/on ";
+
+    /**
+     * prints a line of 100 character length.
+     *
+     */
+    public static void printLine() {
+
+        for(int i = 0; i<100; i++) {
+            System.out.print("-");
+        }
+        System.out.println();
+
+    }
```

Comment:
I like this function as we are using the line everywhere of the iP, extract this as a function is a wise choice. However, maybe 100 '-' is too long?

Reply:
Got it! I will try making the line shorter.